plugins/woocommerce/src/Blocks/BlockTypes/AbstractBlock.php

```diff
@@ -155,7 +155,7 @@ protected function register_block_type_assets() {
 				$this->get_block_type_script( 'handle' ),
 				$this->get_block_type_script( 'path' ),
 				array_merge(
-					$this->get_block_type_script( 'dependencies' ),
+					$data['dependencies'],
```

Comment:
Still working on confirming this, but it looks like we were not enqueueing the dependencies of entry points before (this code just loads an empty array and I don't know why that is). I've changed this to actually merge in the list of dependencies in the asset.php file that is generated per block.

Reply:
Ok I think the reason I saw this was because we cache asset files lol. I was testing in a JN site and installing a new version of same plugin but not clearing transient cache. 

Still curious why this code is like this but I'll look at reverting this.